ci/cloudbuild.yaml

```diff
@@ -31,14 +31,53 @@ steps:
   - '-c'
   - |
     docker build -t gcr.io/$PROJECT_ID/ts-bridge:git-$SHORT_SHA \
-      -t gcr.io/$PROJECT_ID/ts-bridge:build-$BUILD_ID \
+      -t gcr.io/$PROJECT_ID/ts-bridge:build-$build_id \
       -t gcr.io/$PROJECT_ID/ts-bridge:$$(date -u +%Y%m%dT%H%M) \
-      -t gcr.io/$PROJECT_ID/ts-bridge:$$(cat _FULL_TAG) \
+      -t gcr.io/$PROJECT_ID/ts-bridge:$$(cat _release_tag) \
       -t gcr.io/$PROJECT_ID/ts-bridge:$$(cat _MAJOR_TAG) \
       -t gcr.io/$PROJECT_ID/ts-bridge:$$(cat _MINOR_TAG) \
       -t gcr.io/$PROJECT_ID/ts-bridge:latest \
       --cache-from gcr.io/$PROJECT_ID/ts-bridge:latest .
 
+- name: 'docker.io/aquasec/trivy:latest'
+  id: 'Scan newly built image using Trivy and create trivy-out.json'
+  args:
+  - 'image'
+  - '--format=json'
+  - '--output=trivy-out.json'
+  - '--no-progress'
+  - 'gcr.io/cre-tools/ts-bridge:git-$SHORT_SHA'
+
+- name: 'docker.io/aquasec/trivy:latest'
+  id: 'Scan newly built image using Trivy and create trivy-out.table'
+  args:
+  - 'image'
+  - '--output=trivy-out.table'
+  - '--no-progress'
+  - 'gcr.io/cre-tools/ts-bridge:git-$SHORT_SHA'
+
+- name: gcr.io/cloud-builders/gcloud
+  entrypoint: 'bash'
+  args: [ '-c', "gcloud secrets versions access latest --secret=Ts-bridge-bot-token --format='get(payload.data)' | tr '_-' '/+' | base64 -d > git_token.txt" ]
+
+- name: 'python:3'
+  id: 'Parse results and check for vulnerabilities'
+  entrypoint: 'bash'
+  args:
+  - '-c'
+  - |
+    pip install PyGithub
+    pip install absl-py
+    python3 ci/parse-trivy-results.py \
+      --build_id=-$BUILD_ID \
+      --commit_id=$SHORT_SHA \
+      --release_tag=$$(cat _release_tag) \
+      --repo_name=$_REPO_NAME \
+      --token_file=git_token.txt \
+      --trivy_file=trivy-out
+# TODO: change repo name to Google/
```

Comment:
Does this need to happen in this PR?

Reply:
It was kept as the forked branch for testing purposes, I have updated it to be the google branch instead.